test/parallel/parallel.status

```diff
@@ -33,3 +33,7 @@ test-async-hooks-http-parser-destroy: PASS,FLAKY
 [$system==freebsd]
 
 [$system==aix]
+
+[$mode==debug]
+# https://github.com/nodejs/node/issues/28275
```

Comment:
Replace/add https://github.com/nodejs/node/issues/30498?

Reply:
Yeah, #28275 should have been resolved fully.

Reply:
Whoops, yeah, fixed. Thanks.